src/Components/ClustersListTable/ClustersListTable.cy.js

```diff
@@ -199,6 +201,65 @@ describe('data', () => {
   });
 });
 
+const urlParamsList = [
```

Comment:
The test data lacks some cases in which hits are a list of values. E.g. `hits=1,2,3`

Reply:
I made a separate Jira card covering this issue